### PR TITLE
Notebook saved link

### PR DESCRIPTION
**Pull request by nikhilmandlik**
### TODO:
~~- [x] merge `notebook-refactor` branch first PR: https://github.com/nasa/openmct/pull/2883~~
~~- [x] @charlesh88 Need to add style for `click to view`~~

### Author Checklist
Check | Passed
-- | --
Changes address original issue? | Y 
Unit tests included and/or updated with changes? | N/A 
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | Y (Check original issue  #2860)
### Screenshots:
Screen Shot 2020-05-01 at 6 38 39 PM

### CODE REVIEW

**Comment by charlesh88** [comment]
I tweaked the click to view message and pushed that up. LGTM and GTG.

File: src/api/notifications/NotificationAPI.js [hunk 1]
```diff
@@ -75,13 +75,15 @@ export default class NotificationAPI extends EventEmitter {
      * Info notifications are low priority informational messages for the user. They will be auto-destroy after a brief
      * period of time.
      * @param {string} message The message to display to the user
+     * @param {Object} options
```
Comment (deeptailor) [inline src/api/notifications/NotificationAPI.js:78]: Need more info on what this options object can include

File: src/api/notifications/NotificationAPI.js [hunk 2]
```diff
@@ -90,12 +92,14 @@ export default class NotificationAPI extends EventEmitter {
     /**
      * Present an alert to the user.
      * @param {string} message The message to display to the user.
+     * @param {Object} options
```
Comment (deeptailor) [inline src/api/notifications/NotificationAPI.js:95]: same comment as above

File: src/api/notifications/NotificationAPI.js [hunk 3]
```diff
@@ -104,12 +108,14 @@ export default class NotificationAPI extends EventEmitter {
     /**
      * Present an error message to the user
      * @param {string} message
+     * @param {Object} options
      * @returns {Notification}
      */
-    error(message) {
+    error(message, options = {}) {
```
Comment (deeptailor) [inline src/api/notifications/NotificationAPI.js:114]: same comment as above

**Review by deeptailor — CHANGES_REQUESTED** [review]
Just a couple of changes

**Comment by deeptailor** [comment]
@nikhilmandlik - Just have a few questions/comments

**Review by deeptailor — APPROVED** [review]
LGTM

**Comment by deeptailor** [comment]
@nikhilmandlik  - This PR is good to merge, just need to fix some conflicts

**Comment by akhenry** [comment]
@deeptailor  Looks like this is good to merge?

**Review by deeptailor — APPROVED** [review]
LGTM

**Pull request merged**